src/TwigComponent/tests/Fixtures/templates/components/UserCard.html.twig

```diff
@@ -0,0 +1,4 @@
+<div class='user-card'>
+    <p>{{ user.name }}</p>
+    <p>{{ user.email }}</p>
+</div>
```

Comment:
I thought that props that were passed to anonymous components became attributes *unless* you used the `{% props` tag. Did I misunderstand that?

Reply:
I also misunderstood. I think this would make `{{attributes}}` non-stringable like Ryan says(?)

> I thought that props that were passed to anonymous components became attributes unless you used the {% props tag.

Feel like this _should_ be the case as I don't think there's any other way to make the distinction.